collector/database/chart.php

```diff
@@ -267,18 +282,23 @@
 $context['checkboxes'] = $checkboxes;
 $context['metriclabel'] = $context['metriclabel'] ?? get_string('multiplemetrics', 'cltr_database');
 $context['frequency'] = html_writer::empty_tag('input',
-    array('type' => 'hidden', 'name' => 'graphfrequency', 'value' => $selectedfrequency));
+    array('type' => 'hidden', 'name' => 'graphfrequency', 'value' => $displayfrequency));
 $renderer = $PAGE->get_renderer('tool_cloudmetrics');
 
 echo $OUTPUT->header();
 if ($count == 0) {
     echo $OUTPUT->notification(get_string('norecords', 'cltr_database', $maxrecords), 'notifyproblem');
 } else {
+    echo $OUTPUT->notification(get_string('displaying_records', 'cltr_database', ['count' => $count, 'freq' => $freqoptions[$displayfrequency]]), 'notifysuccess');
```

Comment:
We are showing a lot of notifications which are becoming a bit of clutter. Can you swap all of these to 'info' rather than success and can you put them under the graph.

Reply:
Rest is great, once this is tweaked merge away